cedar-policy-core/src/ast/expr.rs

```diff
@@ -144,6 +144,14 @@ pub enum ExprKind<T = ()> {
         /// Be careful the backslash in `\*` must not be another escape sequence. For instance, `\\*` matches a backslash plus an arbitrary string.
         pattern: Pattern,
     },
+    /// Entity type membership. Does the first argument have the entity type
```

Comment:
Nit. Maybe we should say something like "Entity type test" here instead of "Entity type membership" since the latter is often used in schema spec.

Reply:
I like "Entity type test" or "Entity type check"